presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -355,3 +360,75 @@ Lambda Aggregate Functions
         -- (2, 42)
 
     The state type must be a boolean, integer, floating-point, or date/time/interval.
+
+
+Filtering During Aggregation
+----------------------------
+
+The ``FILTER`` keyword can be used to remove rows from aggregation processing
+with a condition expressed using a ``WHERE`` clause. This is evaluated for each
+row before it is used in the aggregation and is supported for all aggregate
+functions.
+
+    AGGREATE_FUNCTION(...) FILTER (WHERE <condition>)
```

Comment:
Lowercase `aggregate_function` since we write functions in lowercase (and keywords in uppercase)

Reply:
I left FILTER and WHERE